src/components/PowerCarouselCard/rawDescriptionToReact.tsx

```diff
@@ -0,0 +1,63 @@
+import { createElement, ReactNode } from "react";
+
+const RAW_DESCRPTION_TAG_NAME_MAP: { [k: string]: string } = { b: "strong", i: "em" };
+
+export function rawDescriptionToReact(rawDescription: string): ReactNode[] {
```

Comment:
NIT: Think it might be good to move this outside the PowerCarouselCard component as we will be using this for the other cards. Definitely can be fixed in a PR when we do this for talents.